src/routes/__tests__/AddCustomDisciplineScreen.spec.tsx

```diff
@@ -67,19 +67,18 @@ describe('AddCustomDisciplineScreen', () => {
     fireEvent.changeText(textField, 'test')
     const submitButton = await findByText(labels.addCustomDiscipline.submitLabel)
     fireEvent.press(submitButton)
-    await waitFor(() => expect(findByText(labels.addCustomDiscipline.error.alreadyAdded)).not.toBeNull())
+    expect(await findByText(labels.addCustomDiscipline.error.alreadyAdded)).not.toBeNull()
   })
 
   it('should show wrong-code-error', async () => {
     const { findByText, findByPlaceholderText } = render(<AddCustomDisciplineScreen navigation={navigation} />, {
       wrapper: wrapWithTheme
     })
-    mocked(loadGroupInfo).mockRejectedValueOnce({ code: 403 })
-
+    mocked(loadGroupInfo).mockRejectedValueOnce({ response: { status: 403 }})
```

Comment:
Not sure why this test worked before ^^

Reply:
Probably because a promise is not `null`. Could we use `queryByText` instead?

Reply:
Only in combination with wait for. However, according to the documentation findByText is the same as waitWor -> getBy, therfore this should not make any difference here